### ISSUE TITLE
python module?

### ISSUE THREAD

**Issue by avaldebe** [metno/fimex#4]
Is there a python module/interface available?

**Comment by alexander-buerger-met-no** [comment]
Yes and no.

Yes, a python interface is being developed. So far it has only very limited functionality.

No, at this moment it is not yet stable enough to make it generally available for testing. It might become available soon, though.

**Comment by trygveasp** [comment]
Hi

Me and Thomas Nipen started on a python interface to NetCDF, which does some things fimex does (quite pragmatic with dimension types etc) 

https://github.com/metno/netcdfpy

Good to know that work is done on a python interface to fimex!

**Comment by cskarby** [comment]
Hello,

I have discussed this and received the following feedback from a python programmer. He asks if we have looked into using boost-python og [pybind](http://pybind11.readthedocs.io/en/stable/index.html)?
The latter is a good match with numpy, which again works well together with netcdf.

As fimex is working on c++/c arrays (that could be exposed via pybind to numpy) one could leverage standard python/netcdf for io-transport. Keeping the algorithms in fimex (only memory) and file io operations in python gives more flexibility on the python side to use other services, eg. if someone has hdf5 instead of netcdf etc. If fimex has readers for other formats that are unique to fimex, those could be exposed as a separate pyfimex.io library and ensure that the output is numpy compatible.
